modules/tableview/index.tsx

```diff
@@ -1,17 +1,34 @@
 import * as React from 'react'
+import {Platform, StyleSheet} from 'react-native'
 import {sectionBgColor} from '@frogpond/colors'
 
 import {
-	TableView,
+	TableView as ActualTableView,
 	Section as ActualSection,
```

Comment:
Rather than aliasing (which I realize is an existing pattern) — thoughts on renaming our wrapper type to avoid confusion? Or, maybe instead of the `Actual` prefix if we instead did something like `RNTableviewSimpleSection` to make clear what its source was?

Reply:
I like the idea of `RNTableviewSimpleSection`. Or even `import * as RNTableView from 'react-native-tableview-simple`; <RNTableView.Section />`?

Reply:
Ooo — I forgot about the second `import * as RNTableView` syntax. I definitely prefer that for namespacing. (It mirrors Rust's `<crate>::<item>` disambiguation form and should be easy to remember.)

The one thing I'm not sure about: `import * as RNTableView` or `import * as RNTableview`?

Reply:
RN does things like `SectionList` and `MaskedViewIOS`, so my vote would be to stick with "*View is a camelcased suffix"?

Reply:
Renamed!